content/applications/marketing/marketing_automation/getting_started/testing_running.rst

```diff
@@ -2,33 +2,45 @@
-them.
+The Odoo *Marketing Automation* app allows users to test marketing campaigns (and mailings) before
+officially running them to check for errors and correct any mistakes before it reaches its target
+audience.
 
-First, open the :menuselection:`Marketing Automation` application, and select the desired campaign,
-which reveals that campaign's detail form.
+Testing campaigns
+=================
 
-On the campaign detail form, make sure the campaign already has activities configured (or build a
-campaign by following the directions here on :doc:`this documentation
+To test a marketing campaign, open the :menuselection:`Marketing Automation` application, and select
+the desired campaign to test, which reveals that campaign's detail form.
+
+On the campaign detail form, make sure the campaign already has activities configured in the
+workflow (or build a campaign by following the directions on :doc:`this documentation
 </applications/marketing/marketing_automation/getting_started/workflow_activities>`).
 
-To start a test, click the :guilabel:`Launch a Test` button at the top of the campaign form.
+To start a test, click the :guilabel:`Launch a Test` button at the top of the campaign form, to the
+right of the :guilabel:`Start` button.
 
 .. image:: testing_running/launch-test.png
    :align: center
-   :alt: Launch a test button in Odoo Marketing Automation.
+   :alt: Launch a test button on a campaign detail form in Odoo Marketing Automation.
+
+When clicked, a :guilabel:`Launch a test` pop-up window appears.
+
+.. image:: testing_running/launch-test-popup-window.png
+   :align: center
+   :alt: Launch a test pop-up window that appears in Odoo Markting Automation.
 
-When clicked, a :guilabel:`Launch a test` pop-up window appears. In the drop-down field, choose an
-existing record (or create a new one) to run the test on. To create a brand new record, click the
-:guilabel:`Search More...` link at the bottom of the drop-down menu, and then click the
-:guilabel:`Create` button.
+Click the field to reveal a drop-down menu of contacts. From this drop-down menu, select an existing
+contact (or create a new one) to run the test on.
 
-Doing so reveals a blank contact form, in which the :guilabel:`Name` and :guilabel:`Email` **must**
-be entered. When all the necessary information has been entered, click :guilabel:`Save`, and Odoo
-returns to the :guilabel:`Launch a test` pop-up, with that new record in the :guilabel:`Choose an
-existing record or create a new one to test` field.
+To create a new contact directly from the :guilabel:`Launch a test` pop-up window, start typing the
+name of the new contact in the blank field, and click :guilabel:`Create and edit...`. Doing so
+reveals a blank :guilabel:`Create Record` pop-up form, in which the necessary contact information
+(:guilabel:`Email`, :guilabel:`Mobile`, etc.) *must* be entered, in order for the test to work.
```

Comment:
```suggestion
To create a new contact directly from the :guilabel:`Launch a test` pop-up window, start typing the
name of the new contact in the blank field, and click :guilabel:`Create and edit...`. Doing so
reveals a blank :guilabel:`Create Record` pop-up form, in which the necessary contact information
(:guilabel:`Email`, :guilabel:`Mobile`, etc.) *must* be entered, in order for the test to work.
```

I would say this action deserves a screenshot and the screenshot in lines 22-24 or Lines 28-30 could be deleted.